tests/test_text_box.py

```diff
@@ -5,14 +5,14 @@
 grid_test = py_cui.grid.Grid(10, 10, 100, 100)
 
 def test_move_right():
-    text_box = py_cui.widgets.TextBox('id', 'Test', grid_test, 1, 1, 1, 2, 1, 0 , 'Hello World')
+    text_box = py_cui.widgets.TextBox('id', 'Test', grid_test, 1, 1, 1, 2, 1, 0 , 'Hello World', password = False)
```

Comment:
One last thing, since these call the `TextBox` initializer, the `password` parameter is actually not a keyword argument. Basically, instead of `password = False` here, it should just be `False`. We would use `password=False` for the `add_text_box` function, but for the initializer its just `password`.

Reply:
Okay this is fixed!